Simplify JMX connector's jmx.dump-tables example

hashhar opened the pull request:
Fixes https://github.com/trinodb/trino/issues/7782

mosabua commented on docs/src/main/sphinx/connector/jmx.rst:
```diff
@@ -32,17 +32,17 @@ To enable periodical dumps, define the following properties:
-History has limited size of ``max-entries`` of entries. Both ``dump-period``
-and ``max-entries`` have default values of ``10s`` and ``86400`` accordingly.
+which MBeans are sampled and stored in memory every ``dump-period``. History
+has a limited size of ``max-entries`` number of entries. Both ``dump-period``
```
Maybe change to break the two properties up in context... something like

```
You can configure the maximum number of history entries with ``max-entries``. It defaults to ``86400``. The time between dumps defaults to ``10s`` and is configured with ``dump-period`` property.
```


The maximum number of entries can be confi history is limited to number of entries, de, ``86400`` by default. It can be configured with ``max-et

mosabua commented on docs/src/main/sphinx/connector/jmx.rst:
```diff
@@ -32,17 +32,17 @@ To enable periodical dumps, define the following properties:
+and ``max-entries`` have default values of ``10s`` and ``86400`` respectively.
 
-Commas in MBean names should be escaped in the following manner:
+Commas in MBean names should be escaped using backslashes (``\``) in the
```
names must be escaped

findepi replied:
here you have one backslash and then you have \\ below
maybe explain that backslaches need to be \-escaped in .properties file?

findepi reviewed (approved):
% other comments

hashhar commented:
@mosabua Please take a look again. Addressed comments. Also added the part about how to split long values into multiple lines (which also gives context about why double-backslashes are required to escape commas) as requested by Piotr.

mosabua reviewed (approved):
Good to go!
